components/LobbyCircle.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from '../styles/home.module.scss';
+
+export interface LobbyCircleProps {
+  name: string;
+  // create options
+  // 0 = not create
+  // 1 = create with no lobbies
+  // 2 = create with lobbies
+  create?: number;
+  offset?: boolean;
+}
+
+export function LobbyCircle({name, create = 0, offset = false}: LobbyCircleProps): JSX.Element {
+  let finalStyle;
+
+  if (create == 1) {
+    finalStyle = `${styles.circle} ${styles.single}`;
+  } else if (create == 2) {
+    finalStyle = `${styles.circle} ${styles.create}`;
+  } else if (offset) {
+    finalStyle = `${styles.circle} ${styles.offset}`;
+  } else {
+    finalStyle = `${styles.circle} ${styles.top}`;
```

Comment:
seems like `styles.circle` is consistent across all these statements so lets move it directly to the JSX element!